src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -2600,7 +2600,7 @@ A catch() block after a catch (System.Exception e) block can catch non-CLS excep
     <value>Invalid preprocessor expression</value>
   </data>
   <data name="ERR_InvalidMemberDecl" xml:space="preserve">
-    <value>Invalid token '{0}' in class, record, struct, or interface member declaration</value>
+    <value>Invalid token '{0}' in class, record, struct, interface, or extension member declaration</value>
```

Comment:
>class, record, struct, interface, or extension member [](http://example.com/codeflow?start=34&length=53)

Would it make sense to simply say "a member" instead of "class, record, struct, interface, or extension member"?